part-2-server/server.js

```diff
@@ -0,0 +1,32 @@
+var htp = require('http')
+var fs = require('fs')
+var server = htp.createServer(function(req, resp){
+    var url = req.url
+    if(url === '/'){
+    	fs.readFile('index.html',function(err,data){
+    		if(err) throw err
+    			else{
+    				resp.write(data.toString())
+    				resp.end()
+    			}
+    	})
+    }
+	else if(url === '/page-2' || url === '/page-3'){
+		fs.readFile('page-2.html', function(err,data){
+			if(err) throw err
+				else{
+					resp.write(data.toString())
+				}
+		})
+	}
+	// else if( url === '/poem'){
+	// 	for (var i = 0 ; i < 139 ; i ++){
```

Comment:
Could either put an `if` statement within a for loop, or break the request.url into two substrings and use the two substrings to respond with the right file